config/config.defaults.yml

```diff
@@ -266,6 +266,9 @@ default: &default_settings
   external_search_url:
   external_search_apikey:
 
+  # Disable Flying Sphinx incremental indexing delayed jobs
+  disable_flying_sphinx_indexing: false
```

Comment:
This should be called `disable_thinking_sphinx_indexing`. Flying sphinx is the cloud service but thinking sphinx is the gem whose actions are controlled by this.